packages/jest-runner/src/JestConfigEditor.ts

```diff
@@ -28,9 +34,19 @@ export default class JestConfigEditor implements ConfigEditor {
     switch (projectType.toLowerCase()) {
       case DEFAULT_PROJECT_NAME:
         return new CustomJestConfigLoader(process.cwd());
+      case 'create-react-app':
+        return new ReactScriptsJestConfigLoader(process.cwd());
+      case 'create-react-app-ts':
+        return new ReactScriptsTSJestConfigLoader(process.cwd());
       case 'react':
+        this.log.warn(
```

Comment:
Is this a place where a fall-through would make sense? So the `react` case only does logging and then "falls through" to the `create-react-app` case? Same for `react-ts` and `create-react-app-ts` of course.

Reply:
Have thought of that, however, I would argue that this is more readable as you don't have to think about which statements do fall-through..

Reply:
Fair enough. Readability is very important!